docs/book/checkout.rst

```diff
@@ -78,7 +78,7 @@ Firstly if the **Customer** is not yet set on the Order it will be assigned depe
 
 .. hint::
 
-    If you are do not understand how Users and Customers work in Sylius got to :doc:`Users Concept documentation </book/users_and_groups>`.
+    If you are do not understand how Users and Customers work in Sylius got to :doc:`Users Concept documentation </book/customer_and_shopuser>`.
```

Comment:
Whole this sense doesn't seem to be right... We should mention, that this concept is related to the shop part only.